File: cs50x/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get size between 1 and 8
    int size;
    do
    {
        size = get_int("Size: ");
    }
    while (size < 1 || size > 8);


    //print blocks
    for(int i = 0; i < size; i++)
    {
        for(int j = 0; j < 2* size + 2; j++)
        {
            if(j < (size - (i + 1)))
            {
                printf(" ");
            }
            else if(j < size)
            {
                printf("#");
            }
            else if (j == size || j == size + 1)
            {
                printf(" ");
            }
            else if(j < (size + 2 + i + 1))
            {
                printf("#");
            }
//            else
//          {
//                printf(" ");
//            }
        }
        printf("\n");
    }
}

File: cs50x/pset6/mario/less/mario.py
from cs50 import get_int

# height between 1 and 8 inclusive
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

# printing piramid
for i in range(height):
    print(" " * (height - 1 - i), "#" * (i + 1), sep="")